app/src/views/private/components/export-sidebar-detail.vue

```diff
@@ -211,6 +211,7 @@
 					<interface-system-fields
 						:value="exportSettings.fields"
 						:collection-name="collection"
+						:can-select-all="true"
```

Comment:
```suggestion
						can-select-all
```

(and rename to `allow-select-all` as well ;) )

Reply:
Fixed 👍